### PR TITLE
Bug password reset

### PR DESCRIPTION
**Pull request by PierrePaul**
https://trello.com/c/gB34pscz/118-password-reset-emails-and-unilingual-say-examplecom-in-subject-line

Subject now includes the environment we are in (example : `[Localhost] Portail Alerte COVID: Réinitialisation du mot de passe`)
Subject line is now translated. The site_name is not though.
The email now fits the figma file.

Screenshot_2020-07-29  Localhost  Portail Alerte COVID Réinitialisation du mot de passe - pierre-paul lefebvre cds-snc ca -
Images should show up when not in localhost.

### CODE REVIEW

File: portal/templates/registration/password_reset_email.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{% load i18n %}
```
Comment (pcraig3) [inline portal/templates/registration/password_reset_email.html:1]: Is this supposed to be an HTML file or a `txt` file?

File: portal/templates/registration/password_reset_email.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+If you did not ask to change your password, ignore this email.
+Thank you,
+
+The COVID Alert team
+
+-------------------------------------------------------------------------
+
+Bonjour {{ user.name }},
+Modification du mot de passe
+Nous avons reçu votre demande de ">modification de mot de passe pour le Portail Alerte COVID.
```
Comment (pcraig3) [inline portal/templates/registration/password_reset_email.html:15]: I think `">` is a mistake

File: portal/templates/registration/password_reset_email_html.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+<p>If you did not ask to change your password, ignore this email.</p>
+<p>Thank you,</p>
+<p>The COVID Alert team</p>
+<br />
+-------------------------------------------------------------------------
+<br />
+<p>Bonjour {{ user.name }},</p>
+<p>Modification du mot de passe</p>
+<p>Nous avons reçu votre demande de <a href="{{ protocol }}://{{ domain }}{% url 'password_reset_confirm' uidb64=uid token=token %}">modification de mot de passe</a> pour le Portail Alerte COVID.</p>
+<p>Vous pouvez modifier votre mot de passe maintenant.</p>
```
Comment (pcraig3) [inline portal/templates/registration/password_reset_email_html.html:16]: In the Figma mockups, the link is on the line below:

Vous pouvez `<a>modifier votre mot de passe</a>` maintenant

File: profiles/templatetags/site_name.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+from profiles.utils import get_site_name
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def site_name(context):
+    request = context.get("request")
+    st = get_site_name(request) + ": "
+    print(st)
```
Comment (pcraig3) [inline profiles/templatetags/site_name.py]: We can just return the `get_site_name(request) + ": "`, right? 

Probably shouldn't be printing this every time.

File: portal/templates/registration/password_reset_email.html (hunk 1 above)
Comment (PierrePaul) [reply portal/templates/registration/password_reset_email.html:1]: This is a txt file.. That's pretty poor naming of the default files. Want me to change the default view to load a file with a better name?

Comment (pcraig3) [reply portal/templates/registration/password_reset_email.html:1]: Nah, if it's the default file then let's just leave it alone.

**Pull request merged**